test/Microsoft.TestPlatform.TestUtilities/TempDirectory.cs

```diff
@@ -51,6 +52,36 @@ public void CopyDirectory(DirectoryInfo source, DirectoryInfo target)
         }
     }
 
+    /// <summary>
+    /// Copy given files into the TempDirectory and return the updated paths that are pointing to TempDirectory.
+    /// </summary>
+    /// <param name="filePaths"></param>
+    /// <returns></returns>
+    public string[] CopyFile(params string[] filePaths)
+    {
+        var paths = new List<string>(filePaths.Length);
+        foreach (var filePath in filePaths)
+        {
+            var destination = IO.Path.Combine(Path, IO.Path.GetFileName(filePath));
+            File.Copy(filePath, destination);
+            paths.Add(destination);
+        }
+
+        return paths.ToArray();
```

Comment:
```suggestion
        return filePaths.Select(path -> CopyFile(path)).ToArray();
```

Reply:
Thought about that, but side-effects in Select are not beautiful. ;)

Reply:
Yep true, so just replacing the content of the foreach by a call to the `CopyFile`.

Reply:
Will do that later, then.